packages/@tinacms/graphql/src/database/store/filesystem.ts

```diff
@@ -0,0 +1,76 @@
+/**
+Copyright 2021 Forestry.io Holdings, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Store } from '.'
+import fs from 'fs-extra'
+import fg from 'fast-glob'
+import path from 'path'
+import normalize from 'normalize-path'
+import { stringifyFile, parseFile } from '../util'
+import { sequential } from '../../util'
+
+export class FilesystemStore implements Store {
+  rootPath: string
+  public async clear() {}
+  public async print() {}
+  constructor({ rootPath }: { rootPath?: string }) {
+    this.rootPath = rootPath || ''
+  }
+  public async query(queryStrings: string[]): Promise<object[]> {
+    throw new Error(`Unable to perform query for Filesystem store`)
+  }
+  public async seed() {
+    throw new Error(`Seeding data is not possible for Filesystem store`)
+  }
+
+  public async get<T extends object>(filepath: string): Promise<T> {
+    return parseFile(
+      await fs.readFileSync(path.join(this.rootPath, filepath)).toString(),
+      path.extname(filepath),
+      (yup) => yup.object()
+    )
+  }
+
+  public supportsSeeding() {
+    return false
+  }
+  public supportsIndexing() {
```

Comment:
The `FilesystemStore` and `GithubStore` aren't really stores, but they're basically taking up the same role as their "bridge" counterparts, it's just easier to inject a store no matter what, and then allow it to define its capabilities. `supportsIndexing` is our way of saying that this store is sort of a fake. These will go away when we're actually ready to ship all of this.